private-templates-service/README.md

```diff
@@ -103,44 +108,20 @@ app.use(function (req, res, next) {
   );
   next();
 });
-`**
-
-**4.  In the `adaptivecards-templates/private-templates-service/server/app.ts` file, replace ` "#{DB_CONNECTION_TOKEN}#"` with the real DB connection string. Make sure to not commit this file.**
-
-**5. Under the `private-templates-service\adaptivecards-templating-service` directory, run `npm link`.**
-
-**6. Under the `adaptivecards-templates/private-templates-service/server` directory, run `npm link adaptivecards-templating-service`, run `npm install`, run `tsc`, and run `npm run start`.**
 
-**You're good to go!**
-
-# Running the frontend
-
-To run this web application locally, you must have the following installed on your system:
-
--   Git
--   [Node v12](https://nodejs.org/en/download/)
-
-**1. Clone the repository.**
-
-**2. Switch to the appropriate branch.**
-
-**3. In the `private-templates-service\adaptivecards-templating-client\typescript\api.ts` file, replace `http://localhost:5000` (defaultBasePath) with `https://adaptivecms.azurewebsites.net`.**
-
-**4.  Under the `private-templates-service\server` directory, run `npm run link-client`.**
-
-**5. Under the `private-templates-service\client` directory, run `npm install` and `npm run start`.**
-
-**You're good to go!**
-
-## Development
-
-**1. Run the command `npm install` in both the `server` and the `client` diretories.**
-**2. Navigate to the `server` folder and run the follwing command `npm run link-client`. If you are getting an error about permission, run the command `sudo npm run link-client`.**
-**3. Inside the `server` folder, run the command `npm run dev`. This will run both the frontend and the backend together. You may now navigate to `localhost:3000` in your browser.**
+```
+5. Replace the MongoDBProvider with the InMemoryProvider. 
+```
+const mongoClient = {
+  authenticationProvider: new AzureADProvider(),
+  storageProvider: new ACMS.InMemoryDBProvider(),
+}
+```
+6. `cd adaptivecards-templates/private-templates-service/server` and run `npm run init-app`. This installs and links dependencies.
 
-## Troubleshooting
+7. Run `npm run dev`. This command concurrently runs the client and server locally. Navigate to `localhost:3000` to see the site.

```

Comment:
Can we say ` In the same directory as the previous step, run npm run dev...` ? 

Just trying to make it 100% clear :)

Reply:
idk if that's overkill....